functests/utils/images/images.go

```diff
@@ -0,0 +1,51 @@
+package images
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+var registry string
+var images map[string]imageLocation
+
+func init() {
+	registry = os.Getenv("IMAGE_REGISTRY")
+
+	images = map[string]imageLocation{
+		"sctp": {
```

Comment:
I don't think we need this one?

Reply:
No, we don't.
I wasn't sure to maintain the same functions equal across the different repos, but they already drifted so I assume it's better to clear.